whois-api/src/test/java/net/ripe/db/whois/api/rest/WhoisRestServiceTestIntegration.java

```diff
@@ -2032,6 +2032,153 @@ public void create_nonauth_asSet_test_aut_num_succed() {
         assertThat(object.getSource().getId(), is("test"));
     }
 
+    @Test
+    public void update_person_add_ripe_ncc_remarks_fails() {
```

Comment:
Can you also add a test that *changes* the end of the remarks: text (after the NCC part) should fail for a non-admin user.